lib/db/schemaupdater.go

```diff
@@ -629,5 +669,298 @@ func (db *schemaUpdater) updateSchemaTo12(_ int) error {
 		return err
 	}
 
+	if err := db.rewriteGlobals(t); err != nil {
+		return err
+	}
+
 	return t.Commit()
 }
+
+func (db *schemaUpdater) rewriteGlobals(t readWriteTransaction) error {
+	it, err := t.NewPrefixIterator([]byte{KeyTypeGlobal})
+	if err != nil {
+		return err
+	}
+	for it.Next() {
+		var vl VersionListDeprecated
+		if err := vl.Unmarshal(it.Value()); err != nil {
+			return err
```

Comment:
should we consider that we might crash after committing some new version lists but before completing the migration, so that we should skip unmarshal errors on the assumption that they might already be rewritten?

Reply:
Adressed by unmarshalling in the new format on error and `continue` if that succeeds.